src/pytest_plugins/consume/consume.py

```diff
@@ -114,6 +161,21 @@ def pytest_addoption(parser):  # noqa: D103
             "The --html flag can be used to specify a different path."
         ),
     )
+    consume_group.addoption(
+        "--sim.limit",
+        action="store",
+        dest="sim_limit",
+        default=None,
+        help=(
```

Comment:
`type=SimLimitBehavior` if we define `process_sim_limit` as the init function for the class.